### PR TITLE
Remote clusterDeployment create options.

### PR DESCRIPTION
**Pull request by jnpacker**
Signed off by: @jnpacker 

* Removed clusterDeployment support for AWS
* Added vendorizing to the build process
* Added Makefile.prow to the Makefile

/cc @itdove @hanqiuzh @leena-jawale

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 -include /opt/build-harness/Makefile.prow
+-include Makefile.prow
```
Comment (hanqiuzh) [inline Makefile:2]: `-include /opt/build-harness/Makefile.prow` twice? and does it work when running locally?

Comment (jnpacker) [reply Makefile:2]: Yeah it did work. Though you make a good point that I don't need the first entry.
I'll try removing it and make sure it works

**Comment by jnpacker** [comment]
Reference: https://github.com/open-cluster-management/backlog/issues/9111

**Pull request merged**